freeipa/src/test/java/com/sequenceiq/freeipa/converter/instance/InstanceGroupRequestToInstanceGroupConverterTest.java

```diff
@@ -87,9 +87,24 @@ void convertTestTemplateConversion() {
         InstanceTemplateRequest instanceTemplateRequest = mock(InstanceTemplateRequest.class);
         request.setInstanceTemplateRequest(instanceTemplateRequest);
         Template template = mock(Template.class);
-        when(templateConverter.convert(instanceTemplateRequest, MOCK, ACCOUNT_ID)).thenReturn(template);
+        when(templateConverter.convert(instanceTemplateRequest, MOCK, ACCOUNT_ID, null)).thenReturn(template);
 
-        InstanceGroup result = underTest.convert(request, ACCOUNT_ID, MOCK.name(), NAME, HOSTNAME, DOMAINNAME);
+        InstanceGroup result = underTest.convert(request, ACCOUNT_ID, MOCK.name(), NAME, HOSTNAME, DOMAINNAME, null);
+
+        assertThat(result).isNotNull();
+        assertThat(result.getTemplate()).isSameAs(template);
+    }
+
+    @Test
+    void convertTestTemplateConversionWithDiskEncryptionSetId() {
```

Comment:
Please add a similar case for the `createDefaultTemplate()` path when having `"dummyDiskEncryptionSetId"`.